src/test/app/common/country.ts

```diff
@@ -0,0 +1,39 @@
+/* tslint:disable:no-unused-expression */
+
+import * as chai from 'chai'
+import * as chaiAsPromised from 'chai-as-promised'
+chai.use(chaiAsPromised)
+
+import { Country } from 'app/common/country'
+
+describe('Country', () => {
```

Comment:
there is much more logic to cover in tests in `app/common/country` file.

Reply:
I couldn't figure out how to mock returning Promises :(

Reply:
the way we do it in unit tests for routes wouldn't work here?